shared/django_apps/core/migrations/0053_useronboardinglifecyclemetrics.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.13 on 2024-06-07 14:57
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
```

Comment:
Could you pls add the sql comment here? See other migrations for reference, you'd need to run `python manage.py sqlmigrate <app> <number>`; `python manage.py sqlmigrate core 0053`